features/employers/benefit_applications/admin_revise_end_date_ben_app_steps.rb

```diff
@@ -11,3 +11,31 @@
 Then(/^the user will (.*) Revise End Date button$/) do |action|
   action == 'see' ? (page.has_css?('Revise End Date') == true) : (page.has_css?('Revise End Date') == false)
 end
+
+And(/^initial employer ABC Widgets has updated (.*) effective period for revise end date$/) do |aasm_state|
+  step "initial employer ABC Widgets has updated #{aasm_state} effective period for reinstate"
```

Comment:
if you can revert changes to step name as stated above, this is not required.

Reply:
Mentioned above the reason to change it.